webkit2/webkit2.init.lisp

```diff
@@ -26,12 +26,14 @@
 
 (defcfun "webkit_get_major_version" :int)
 (defcfun "webkit_get_minor_version" :int)
+(defcfun "webkit_get_micro_version" :int)
 
 (eval-when (:load-toplevel :execute)
   (with-standard-io-syntax
-    (let* ((version (format nil "~a.~a"
+    (let* ((version (format nil "~a.~a.~a"
```

Comment:
While we're at that, I'd add several features here (resembling Nyxt):
- Major version.
- Major+minor version.
- Full version.

Reply:
For WebKitGTK 2.40.1 we push three features:
- `WEBKIT-2` (already pushed as `WEBKIT2` instead).
- `WEBKIT-2.40` (pushed previously, not pushed in this PR).
- `WEBKIT-2.40.1` (pushed in this PR).